### PR TITLE
feat: proper ETH conversion on CloseTrade

### PR DESCRIPTION
**Pull request by MeLlamoPablo**

### CODE REVIEW

File: apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx [hunk 1]
```diff
@@ -32,10 +32,10 @@ const ClosePositionButton: React.FC = () => {
     if (
       closePositionValueBN !== undefined &&
       closePositionValueBN.gt(0) &&
```
Comment (XavierEkkel) [inline apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx:34]: if there are both ETH and wstETH values in a function, it'd be clearer to indicate which ones are ETH vs wstETH
```suggestion
      closePositionValueInEthBN !== undefined &&
      closePositionValueInEthBN.gt(0) &&
```

File: apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx [hunk 2]
```diff
@@ -32,10 +32,10 @@ const ClosePositionButton: React.FC = () => {
...
-      selectedPosition.totalValueBN !== undefined
+      selectedPosition.totalValueInETHBN !== undefined
```
Comment (XavierEkkel) [inline apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx:35]: nit: use camel case for Eth for readability
```suggestion
      selectedPosition.totalValueInEthBN !== undefined
```

File: apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx (hunk 1 above)
Comment (MeLlamoPablo) [reply apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx:34]: I wanted to minimize changes to existing code, I can take a pass at this on another PR

File: apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx (hunk 2 above)
Comment (MeLlamoPablo) [reply apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx:35]: Same, these were introduced in previous PRs so I didn't want to touch existing code

**Review by chrisling-dev — APPROVED** [review]
LGTM

**Pull request merged**